src/assets/i18n/en.json5

```diff
@@ -5434,17 +5430,11 @@
 
   "access-control-remove": "Remove access condition",
 
-  "access-control-select-bitstreams-modal.title": "Select bitstreams",
-
-  "access-control-select-bitstreams-modal.no-items": "No items to show.",
-
-  "access-control-select-bitstreams-modal.close": "Close",
-
-  "access-control-option-label": "Access condition type",
+  "community.search.head": "Search Within",
 
-  "access-control-option-note": "Choose an access condition to apply to selected objects.",
+  "collection.page.browse.search.head": "Search Within",
```

Comment:
This i18n key doesn't seem to be used in this PR?  I think it can be removed maybe?